x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/index.test.tsx

```diff
@@ -359,4 +365,29 @@ describe('StepAboutRuleComponent', () => {
       expect(result2?.data?.riskScore.value).toEqual(47);
     });
   });
+
+  it('should use index based on ML jobs when editing ML rule', async () => {
+    (useFetchIndex as jest.Mock).mockClear();
+    useGetInstalledJobMock.mockImplementation((jobIds: string[]) => {
+      expect(jobIds).toEqual(['auth_high_count_logon_events_for_a_source_ip']);
+      return { jobs: [{ results_index_name: 'shared' }] };
+    });
+
+    mount(
+      <ThemeProvider theme={mockTheme}>
+        <StepAboutRule
+          addPadding={true}
+          defaultValues={stepAboutDefaultValue}
+          defineRuleData={stepDefineStepMLRule}
+          descriptionColumns="multi"
+          isReadOnlyView={false}
+          setForm={setFormHook}
+          isLoading={false}
+        />
+      </ThemeProvider>
```

Comment:
Nit: we have a `TestProviders` component that contains all common providers required for tests, including `ThemeProvider`. You can pass it as a wrapping component to `mount`:

```ts
mount(<YourComponent />, {
  wrappingComponent: TestProviders,
});
```

That approach is a bit more future-proof, as you wouldn't need to adjust your tests when the internal implementation of the tested component is changed and, let's say, a new provider dependency gets added.

Reply:
done